scripts/jenkins/groovy/buildConfig.groovy

```diff
@@ -336,12 +336,15 @@ class BuildConfig {
 
   static enum JenkinsMaster {
     C1, // indicates we are running under mr-0xc1 master - master or nightly build
+    G1, // indicates we are running under mr-0xg1 master - master or nightly build
     B4  // indicates we are running under mr-0xb4 master - PR build
 
     private static JenkinsMaster findByName(final String name) {
       switch(name.toLowerCase()) {
         case 'c1':
```

Comment:
do we need separate settings for both? I think we can just merge these two case: into one and have only one setting

Reply:
I agree

Reply:
I would like to see it running now and clean it (the usage of enum is a bit of overkill here, IMO and can be cleaned)